gcp/modules/locust/client.rb

```diff
@@ -0,0 +1,71 @@
+require "csv"
```

Comment:
(Since it seems that PR comments not associated with a line of code often get lost, I'm attaching this comment here arbitrarily.)

> It is manually created, there is no way to manage it in code – no API for dashboards yet

Can you link me to the documentation for how to manually create this dashboard? I guess we'll need to repeat this process in long-lived environments, and repeat it each time we want to view these metrics in short-lived environments?

Reply:
This is being addressed in https://github.com/gpii-ops/gpii-infra/pull/195.